### PR TITLE
Fix people you may know not working

### PR DESCRIPTION
**Pull request by RUPESH1439**

### CODE REVIEW

File: frontend/src/api/UserRequest.js [hunk 1]
```diff
@@ -1,19 +1,21 @@
-import axios from 'axios';
+import axios from "axios";
```
Comment (RUPESH1439) [inline frontend/src/api/UserRequest.js:1]: Can we not use double quote here? 
@RUPESH1439

**Review by RUPESH1439 — COMMENTED** [review]
Looks good to me!

File: frontend/src/api/UserRequest.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-API.interceptors.request.use((req)=>{
-    if(localStorage.getItem('profile')){
-        req.headers.Authorization = `Bearer ${JSON.parse(localStorage.getItem('profile').token)}`
-    }
-    return req
-})
+API.interceptors.request.use((req) => {
+  if (localStorage.getItem("profile")) {
+    req.headers.Authorization = `Bearer ${
+      JSON.parse(localStorage.getItem("profile")).token
```
Comment (delfina-ghimire) [inline frontend/src/api/UserRequest.js:10]: Looks good, thanks! 😊

**Review by delfina-ghimire — APPROVED** [review]
COOOL! 😍

**Pull request merged**